ui/packages/theme/src/tokens.css

```diff
@@ -112,7 +112,7 @@
 	.gr-sample-dataframe,
 	.gr-sample-audio,
 	.gr-sample-3d {
-		@apply flex items-center border cursor-pointer px-2 py-1.5 rounded-lg bg-white hover:bg-gray-50 text-sm text-left dark:hover:bg-gray-800;
+		@apply flex items-center border cursor-pointer px-2 py-1.5 rounded-lg bg-white hover:bg-gray-50 text-sm text-left dark:hover:bg-gray-800 dark:bg-transparent;
```

Comment:
Looks good to me. I was able to verify it fixed the original problem and I also tried with single image/video/audio example demos.

@pngwn @dawoodkhan82 @aliabid94 Does this change make sense to you?